virtcontainers/cgroups.go

```diff
@@ -0,0 +1,111 @@
+// +build linux
```

Comment:
Could you remove this comment and just rename the file to be `virtcontainers/cgroups_linux.go` as that's much clearer imho.

Reply:
And, you need a new file named "virtcontainers/cgroups_unsupported.go" contains exactly same interface with current implementation(old codes) for other platforms other than linux. Or it won't compile on ppc64le

Reply:
You can imitating 
```
 vendor/github.com/opencontainers/runc/libcontainer/cgroups/cgroups_unsupported.go
@@ -0,0 +1,3 @@
 +// +build !linux
 +
 +package cgroups
```